docs/pages-for-subheaders/installation-requirements.md

```diff
@@ -55,68 +55,126 @@ If you are installing Rancher on a K3s cluster with **Raspbian Buster**, follow
 
 If you are installing Rancher on a K3s cluster with Alpine Linux, follow [these steps](https://rancher.com/docs/k3s/latest/en/advanced/#additional-preparation-for-alpine-linux-setup) for additional setup.
 
-### RKE2 Specific Requirements
+### RKE Specific Requirements
 
-For the container runtime, RKE2 bundles its own containerd. Docker is not required for RKE2 installs.
+For the container runtime, RKE should work with any modern Docker version.
 
-For details on which OS versions were tested with RKE2, refer to the [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions).
+For more information, see [Installing Docker](../getting-started/installation-and-upgrade/installation-requirements/install-docker.md).
 
 ## Hardware Requirements
 
-The following sections describe the CPU, memory, and disk requirements for the nodes where the Rancher server is installed.
+The following sections describe the CPU, memory, and I/O requirements for nodes where Rancher is installed. Requirements vary based on the size of the infrastructure.
 
-## CPU and Memory
+### Practical Considerations
 
-Hardware requirements scale based on the size of your Rancher deployment. Provision each individual node according to the requirements. The requirements are different depending on if you are installing Rancher in a single container with Docker, or if you are installing Rancher on a Kubernetes cluster.
+Rancher's hardware footprint depends on a number of factors, including:
 
-### RKE and Hosted Kubernetes
+ - Size of the managed infrastructure (e.g., node count, cluster count).
+ - Complexity of the desired access control rules (e.g., `RoleBinding` object count).
+ - Number of workloads (e.g., Kubernetes deployments, Fleet deployments).
+ - Usage patterns (e.g., subset of functionality actively used, frequency of use, number of concurrent users).
 
-These CPU and memory requirements apply to each host in the Kubernetes cluster where the Rancher server is installed.
+Since there are a high number of influencing factors that may vary over time, the requirements listed here should be understood as reasonable starting points that work well for most use cases. Nevertheless, your use case may have different requirements. For inquiries about a specific scenario please [contact Rancher](https://rancher.com/contact/) for further guidance.
 
-These requirements apply to RKE Kubernetes clusters, as well as to hosted Kubernetes clusters such as EKS.
+In particular, requirements on this page are subject to typical use assumptions, which include:
+ - Under 60 thousand total Kubernetes resources, per type.
```

Comment:
minor note - maybe change to 60,000 for consistency

Reply:
Addressed as of d5b649a6ca9